cloud/cloud.py

```diff
@@ -1,18 +1,24 @@
+import libcloud
+import toml
+
 import cloud
 from cloud import registry as reg
 from cloud import Instance
 
+instance = None
+
+
+def connect(path):
```

Comment:
I don't think the use should have to pass a path each time. We should have some hierarchy of where we look for a `cloud.config` file:

1. In $CLOUD_CFG
2. In "/"
3. Passed to connect()